run.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/#!/usr/bin/env bash
+
+docker run -it --rm --name tester docker-registry.cluster.fravega.com/fravega/rabbit-mq-stress-tester:1.1.2 ./rabbit-mq-stress-tester --help
```

Comment:
Did you mean to commit this as well? I don't know if this docker image should be used by everyone.